src/networking.c

```diff
@@ -373,6 +377,15 @@ void deleteCachedResponseClient(client *recording_client) {
     freeClient(recording_client);
 }
 
+static void resetBlockedClientPendingReply(client *c) {
+    c->blocked_client_pending_reply_bytes = 0;
+    if (c->blocked_client_pending_reply == NULL) {
+        return;
+    }
+    listRelease(c->blocked_client_pending_reply);
+    c->blocked_client_pending_reply = NULL;
```

Comment:
```suggestion
    listRelease(c->blocked_client_pending_reply);
    c->blocked_client_pending_reply = NULL;
```
Most Valkey functions gracefully handle freeing NULL pointers.